### ISSUE TITLE
License

### ISSUE THREAD

**Issue by akutheman** [Joohansson/NanoRPCProxy#5]
The switch to GPL means a lot of people can no longer use NanoRPCProxy. What was the rationale for this switch?

The entire Nano ecosystem uses permissive BSD and MIT licenses to avoid all the issues with GPL and proprietary system (which some Nano-based companies necessarily will be)

Using the REST api only is probably fine (although, call a lawyer, this is GPL infestation after all), but my usecase means embedding the server in a process container, which means GPL seeps into my system (I checked this with a different GPL-ed server)

Also, anyone forking NanoRPCProxy to add tweaks to a closed-sourced server no longer can. Is this the intention?

**Comment by Joohansson** [comment]
To be honest, I switched to GPL because I realized I didn't like the idea of MIT and that anyone can close source, make it their own proprietary work and sell it however they want. Given how much the Nano community like projects to be open source I thought this was going to be met only with positivity. I could never have imagined it would be a problem for this kind of "open" project as you describe it, so that comes as a complete surprise. I'm sorry it causes problems for you but it just the way it's going to be. To answer your last question, yes it was the intention. I think all my other repositories are MIT and will probably stay that way.
